draft-ietf-jsonpath-base.md

```diff
@@ -963,7 +960,7 @@ FUNCTION Normalize(i, len):
   END IF
 ~~~~
 
-The result of the array indexing expression `i` applied to an array
+The result of the array index expression `i` applied to an array
```

Comment:
Well, this isn't a pure array index.
I think we still want to distinguish the index (0..n-1) from indexing (which, for instance, can take -2).

Reply:
I'll take another look at this PR with that distinction in mind and see how it goes.

Reply:
@cabo: re-reading the spec with this PR applied (and after tightening up the semantics of the index selector), I think the generalised index terminology works ok. An index is now defined to be "An integer that identifies a specific element in an array" which includes the case of negative indices. So I don't think we need the rather fine distinction between index and indexing (this term is no longer used). Please could you take a fresh look and see if you agree.